ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -14,6 +14,12 @@ function populate_artifact_dir() {
     s/X-Auth-Token.*/X-Auth-Token REDACTED/;
     s/UserData:.*,/UserData: REDACTED,/;
     ' "${dir}/.openshift_install.log" > "${ARTIFACT_DIR}/.openshift_install.log"
+  sed '
+    s/password: .*/password: REDACTED/;
+    s/X-Auth-Token.*/X-Auth-Token REDACTED/;
+    s/UserData:.*,/UserData: REDACTED,/;
+    ' "${dir}/terraform.log" > "${dir}/terraform.log"  
+  tar -czvf "${dir}/terraform.tar.gz" --remove-files "${dir}/terraform.txt" 
```

Comment:
```suggestion
  tar -czvf "${ARTIFACT_DIR}/terraform.tar.gz" --remove-files "${dir}/terraform.txt" 
```
So it's actually saved when the artifacts dir is copied.